### PR TITLE
added utility styling

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added utility classes for colors,typography,grid and some frequently used css property like diplay,z-index,position,width,height,padding,margin etc.

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
code could be more readable... 👍

File: css/util/color/index.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+.bg-primary {
+    background-color: var(--primary-color)  !important;
```
Comment (deekshithmd) [inline css/util/color/index.css]: as vikram mentioned don't use !important unless it is very much required

File: css/util/typography/index.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  .text-justify{
+    text-align: justify;
+  }
+
+  .text-md-bold{
+    font-weight: 400;
+  }
+  .text-bolder{
+    font-weight: bolder;
+  }
```
Comment (deekshithmd) [inline css/util/typography/index.css]: try to give line gap between classes

File: css/util/color/index.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/util/color/index.css]: Okay!
I will make changes to that.

File: css/util/typography/index.css (hunk 2 above)
Comment (sanjitsarkar) [reply css/util/typography/index.css]: Okay.
I will fix this

File: css/util/color/index.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/util/color/index.css]: without !important breaks the layout on card component.

Comment (sanjitsarkar) [reply css/util/color/index.css]: Could you approve this

**Review by deekshithmd — APPROVED** [review]

File: css/util/color/index.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/util/color/index.css]: Fixed the breaking of layout without using !important

**Pull request merged**